tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -328,10 +342,9 @@ kbs_k8s_svc_host() {
 	if kubectl get ingress -n "$KBS_NS" 2>/dev/null | grep -q kbs; then
 		kubectl get ingress kbs -n "$KBS_NS" \
 			-o jsonpath='{.spec.rules[0].host}' 2>/dev/null
-	elif kubectl get svc kbs-nodeport -n "$KBS_NS" &>/dev/null; then
+	elif kubectl get svc kbs -n "$KBS_NS" &>/dev/null; then
```

Comment:
This was hardcoded as it was nodeport-related only. Now we should simply get from `$KBS_SVC_NAME` to avoid rename issues.

Reply:
changes applied